docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -222,25 +227,150 @@ Iceberg. In addition to the :ref:`globally available <sql-globally-available>`
 and :ref:`read operation <sql-read-operations>` statements, the connector
 supports the following features:
 
-* :doc:`/sql/insert`
-* :doc:`/sql/delete`, see also :ref:`iceberg-delete`
-* :doc:`/sql/update`
-* :doc:`/sql/merge`
-* :ref:`sql-schema-table-management`, see also :ref:`iceberg-tables`
-* :ref:`sql-materialized-view-management`, see also
-  :ref:`iceberg-materialized-views`
-* :ref:`sql-view-management`
+* :ref:`sql-write-operations`:
+
+  * :ref:`iceberg-schema-table-management` and :ref:`iceberg-tables`
+  * :ref:`iceberg-data-management`
+  * :ref:`sql-view-management`
+  * :ref:`sql-materialized-view-management`, see also :ref:`iceberg-materialized-views`
+
+.. _iceberg-schema-table-management:
+
+Schema and table management
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :ref:`sql-schema-table-management` functionality includes support for:
+
+* :doc:`/sql/create-schema`
+* :doc:`/sql/drop-schema`
+* :doc:`/sql/alter-schema`
+* :doc:`/sql/create-table`
+* :doc:`/sql/create-table-as`
+* :doc:`/sql/drop-table`
+* :doc:`/sql/alter-table`
+* :doc:`/sql/comment`
+
+.. _iceberg-create-schema:
+
+CREATE SCHEMA
+~~~~~~~~~~~~~
+
+The connector supports creating schemas. You can create a schema with or without
+a specified location.
+
+You can create a schema with the :doc:`/sql/create-schema` statement and the
+``location`` schema property. The tables in this schema, which have no explicit
+``location`` set in :doc:`/sql/create-table` statement, are located in a
+subdirectory under the directory corresponding to the schema location.
+
+Create a schema on S3::
+
+  CREATE SCHEMA iceberg.my_s3_schema
+  WITH (location = 's3://my-bucket/a/path/');
+
+Create a schema on a S3 compatible object storage such as MinIO::
+
+  CREATE SCHEMA iceberg.my_s3a_schema
+  WITH (location = 's3a://my-bucket/a/path/');
+
+Create a schema on HDFS::
+
+  CREATE SCHEMA iceberg.my_hdfs_schema
+  WITH (location='hdfs://hadoop-master:9000/user/hive/warehouse/a/path/');
+
+Optionally, on HDFS, the location can be omitted::
+
+  CREATE SCHEMA iceberg.my_hdfs_schema;
+
+.. _iceberg-create-table:
+
+Creating tables
+~~~~~~~~~~~~~~~
+
+The Iceberg connector supports creating tables using the :doc:`CREATE
+TABLE </sql/create-table>` syntax. Optionally specify the
+:ref:`table properties <iceberg-table-properties>` supported by this connector::
+
+    CREATE TABLE my_table (
+        c1 integer,
+        c2 date,
+        c3 double
```

Comment:
nit: Type names are lowercase here, but the following DDL for NOT NULL example uses uppercase.